scripts/run-android.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+set -m # needed to access jobs
+
+GIT_ROOT=$(cd "${BASH_SOURCE%/*}" && git rev-parse --show-toplevel)
+
+# generates android debug build when we pass the debug flag
+"${GIT_ROOT}"/scripts/build-android.sh
+
+# installs the resulting apk on running emulator or android device
+adb install ./result/app-debug.apk
+
+# start metro and go to background
+nohup "${GIT_ROOT}"/scripts/start-react-native.sh 2>&1 | tee metro-server-logs.log &
```

Comment:
Shouldn't we add `metro-server-logs.log` to `.gitignore`?

Reply:
I didn't do that since its already been caught by some pattern inside `.gitignore`

My guess is this one : 
https://github.com/status-im/status-mobile/blob/develop/.gitignore#L190

Reply:
Happy to make it more explicit :)